packages/rest-specs/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@typespec/rest-specs",
```

Comment:
I know Brian suggested that but I think `http-specs` makes more sense as the name as it is testing a lot of the lower level http functionalities and that necessary tied to rest

Reply:
Done